eng/packaging.props

```diff
@@ -27,6 +27,7 @@
   </PropertyGroup>
 
   <PropertyGroup>
+    <NuSpecOutputPath Condition="'$(NuSpecOutputPath)' == ''">$([MSBuild]::NormalizeDirectory('$(ArtifactsPackagesDir)', 'specs'))</NuSpecOutputPath>
```

Comment:
Why do we set this for libraries packages only? Should this be moved into the root Directory.Build files instead?

Do we need the condition?

Reply:
Currently they donot produce that, we can fix in that in a separate change.

Reply:
If for other packages we don't need the nuspec in a specific place, I'm fine with that. Out of curiosity, what are using the nuspec for? Package validation?

Reply:
i dont think we need them in a specific place place, they are not used in anything

Reply:
Then why is this change necessary? Can't we just have them in their default location which I believe is in their intermediate output directories?

Reply:
> Then why is this change necessary? Can't we just have them in their default location which I believe is in their intermediate output directories?

just to keep the things similar to the other nuspecs, we can move them all together to their default locations

Reply:
Makes sense, let's post pone this.